brototype.js

```diff
@@ -140,15 +142,7 @@
         }
     };
 
-    (function() {
-        if (typeof define === 'function' && typeof define.amd === 'object') {
-            define(function() {
-                return Bro;
-            });
-        } else if (typeof module !== 'undefined' && module.exports) {
-            module.exports = Bro;
-        } else if (typeof window !== 'undefined') {
-            window.Bro = Bro;
-        }
-    })();
```

Comment:
require and window work because of Bro being moved out of the closure above.  I will put together some examples